### ISSUE TITLE
share logs of pretrain

### ISSUE THREAD

**Issue by itsliupeng** [jackroos/VL-BERT#10]
Excellent work!
I used the script and https://github.com/jackroos/VL-BERT/blob/master/cfgs/pretrain/base_e2e_16x16G_fp16.yaml to pretrain，but can't be sure if the loss and accuracy are good enough.
When using my pre-trained model to finetune in VCR task, I cannot get the high score of using the shared pre-trained model.
I will appreciate for your share of pre-trained logs. 
Thanks.

**Comment by jackroos** [comment]
@itsliupeng Could you please share your training log? Thanks!

**Comment by itsliupeng** [comment]
> And are you using 16 GPUs to run this config?

Using 3 * 8 = 24 V100 GPUs,   BATCH_IMAGES is [16, 16], others are same with https://github.com/jackroos/VL-BERT/blob/master/cfgs/pretrain/base_e2e_16x16G_fp16.yaml

**Comment by jackroos** [comment]
@itsliupeng Maybe the learning rate is too larger since the code would set the learning rate linearly w.r.t batch size and the optimizer is Adam instead of SGD.

**Comment by itsliupeng** [comment]
> @itsliupeng Maybe the learning rate is too large since the code would set the learning rate linearly w.r.t batch size and the optimizer is Adam instead of SGD. So you may need to tune the learning rate if you change the batch size.

Thanks for relaying. I will try it using all the default parameters.

following are the metrics after 10 epochs:

Train-MLMAccWVC=0.602038,	MLMAccAUX=0.481264,	MVRCAccuracy=0.699305,	MLMLossWVC=1.938624,	MLMLossAUX=3.015435,	MVRCLoss=2.003631,

Val-MLMAccWVC=0.577410,	MLMAccAUX=0.443705,	MVRCAccuracy=0.680914,	MLMLossWVC=2.092122,	MLMLossAUX=3.141392,	MVRCLoss=2.088244,

**Comment by jackroos** [comment]
The pre-training loss is too high and acc is too low. You can use default settings to reproduce our result. And you should check the training process by tensorboard to see if the training is collapsed (the training loss suddenly increases), it happens occasionally.

**Comment by jackroos** [comment]
Hi, I just found the max box number is wrong in our shared repo for extracting rois for Conceptual Captions. Maybe it would affect the pre-training. You can refer to ![this issue](https://github.com/jackroos/VL-BERT/issues/19) to modify the max box number. Sorry for this mistake!

**Comment by itsliupeng** [comment]
> @itsliupeng Hi, I just found the max box number is wrong in our shared repo for extracting rois for Conceptual Captions. Maybe it would affect the pre-training. You can refer to [this issue](https://github.com/jackroos/VL-BERT/issues/19) to modify the max box number. Sorry for this mistake!

Thanks for your informed.
